utils/spacewalk-common-channels.ini

```diff
@@ -1002,14 +1011,23 @@ base_channels = ubuntu-16.04-pool-amd64-uyuni
 repo_url = http://archive.ubuntu.com/ubuntu/dists/xenial-security/main/binary-amd64/
 
 [ubuntu-1604-amd64-universe-uyuni]
-label    = ubuntu-1604-amd64-main-universe-uyuni
```

Comment:
Same for this one.

Reply:
Doc is correct.

Reply:
ACK, did you also check that the new channels for universe updates are added to the doc?